app/views/works/index.html.erb

```diff
@@ -0,0 +1,93 @@
+
+ <h2>List of Works</h2>
+
+ <section class="media-table">
+   <h4>Albums</h4>
+   <table class="table">
+     <thead>
+       <tr>
+         <th scope="col" width>Votes</th>
+         <th scope="col" >Title</th>
+         <th scope="col" >Created By</th>
+         <th scope="col" width>Published</th>
+         <th scope="col" width>Upvote</th>
+       </tr>
+     </thead>
+
+     <tbody>
+       <% @works.each do |work| %>
+       <% if work.category == 'album' %>
```

Comment:
Is there a way we could reduce the repeated code here that checks on the work's category? This is essentially duplicated code except for the h4 and the category.